### ISSUE TITLE
<Urgent> Downloading libbarhopper.so 

### ISSUE THREAD

**Issue by onuryurtturk** [googlesamples/android-vision#400]
Is there any way to download libbarhopper.so file externally? Many of our users have problems with stabilization about barcode reading. Sometimes it doesn't work after reboot sometimes works after application restart.  Although we use the current version of Google play services, we are experiencing this problem on different devices at different times.

We have about 50k users and we often get feedback from users on this issue. We need your return for immediate solution.

Used Devices : Samsung  J7, XCover3 and Galaxy S4

**Comment by onuryurtturk** [comment]
@theoreticalb @claywilkinson @pm0733464 @pchx @omares @Gericop @incon @bhargavms @mbg1 @AdamSHurwitz @paolorotolo @ragdroid @Morion-Self @gonzapico

**Comment by onuryurtturk** [comment]
Should we try ML Kit as a solution? Does the ML Kit have stabilization problems?

@theoreticalb @claywilkinson @pm0733464 @pchx @omares @Gericop @incon @bhargavms @mbg1 @AdamSHurwitz @paolorotolo @ragdroid @Morion-Self @gonzapico

**Comment by onuryurtturk** [comment]
https://issuetracker.google.com/issues/80454351
https://issuetracker.google.com/issues/109900815

**Comment by onuryurtturk** [comment]
@theoreticalb @claywilkinson @pm0733464 @pchx @omares @Gericop @incon @bhargavms @mbg1 @AdamSHurwitz @paolorotolo @ragdroid @Morion-Self @gonzapico

**Comment by onuryurtturk** [comment]
@theoreticalb @claywilkinson @pm0733464 @pchx @omares @Gericop @incon @bhargavms @mbg1 @AdamSHurwitz @paolorotolo @ragdroid @Morion-Self @gonzapico

**Comment by onuryurtturk** [comment]
@theoreticalb @claywilkinson @pm0733464 @pchx @omares @Gericop @incon @bhargavms @mbg1 @AdamSHurwitz @paolorotolo @ragdroid @Morion-Self @gonzapico

**Comment by Gericop** [comment]
@onuryurtturk Stop tagging people. Wait for the issues to be fixed.

**Comment by onuryurtturk** [comment]
We have been waiting solution for 6 months! I'm going to keep tagging unless I receive a reasonable explanation @Gericop 

Check out : issue tracking messages

https://issuetracker.google.com/issues/80454351
https://issuetracker.google.com/issues/109900815

**Comment by Gericop** [comment]
@onuryurtturk You do understand that most of the people you tag here are probably **not** Google devs, right?

**Comment by onuryurtturk** [comment]
@Gericop  Yeah, maybe. There is no announcements about this situation neither on issue tracker nor here for a long time. I can't wait long, so I'm trying my best to fix the situation before giving up the library. We are losing user because of this issue. Well, did you understand?....

**Comment by Gericop** [comment]
@onuryurtturk I understand that you have issues with it, but still. This is just a sample project for the Mobile Vision API, not an actual library project. Stop spamming...
